example-projects/advanced/phantom/repl.js

```diff
@@ -1,10 +1,23 @@
-if (phantom.args.length != 1) {
-    console.log('Expected a target URL parameter.');
-    phantom.exit(1);
+var system = require('system');
+var url;
+
+if (phantom.version.major > 1) {
+    var args = system.args;
```

Comment:
Same as the url, could you move the `args` declaration outside the if statement, e.g. `var url, args;`